### PR TITLE
Remove unnecessary h2database version

### PR DESCRIPTION
**Pull request by cuijulian**
Depends on https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1098

### CODE REVIEW

**Comment by cuijulian** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
It seems like MVCC has been enabled by default since version 1.4.200 which provokes 
Unsupported connection setting "MVCC" [90113-200] errors in lots of tests

Here, they suggest just dropping the MVCC param from the JDBC URL:
https://groups.google.com/forum/#!topic/h2-database/93i3fA1BF08

**Comment by cuijulian** [comment]
jenkins retest this

**Comment by mareknovotny** [comment]
There are still problems with creating connection on all failed tests, please try to fix it.

`Caused by: java.sql.SQLException: Cannot create PoolableConnectionFactory (A file path that is implicitly relative to the current working directory is not allowed in the database URL "jdbc:h2:tcp://localhost/target/jbpm-bpmn-db". Use an absolute path, ~/name, ./name, or the baseDir setting instead. [90011-200])`

**Comment by cuijulian** [comment]
@mareknovotny the SQLException above can be solved by allowing relative paths. However, there is another issue that is uncovered: the latest version of h2 includes security patches that enable `;IFEXISTS=true` by default, which cannot be configured in the path URL (see this [issue](https://github.com/h2database/h2database/issues/1901)).

Thus, an `org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database "/home/jucui/kiegroup/jbpm/jbpm-persistence/jbpm-persistence-jpa/target/jbpm-test" not found, either pre-create it or allow remote database creation (not recommended in secure environments) [90149-200]` is thrown during tests. According to the [h2 docs](https://h2database.com/html/tutorial.html#creating_new_databases), this can only be fixed by allowing remote database creation, which is done by passing the `-ifNotExists` parameter in the command line.

In short, I'm stuck. I don't see a trivial way to do this in the code. Is there something I'm missing?

**Comment by cuijulian** [comment]
Jenkins retest this

**Comment by mareknovotny** [comment]
can you @cuijulian rebase?

**Comment by cuijulian** [comment]
Jenkins retest this

**Comment by mareknovotny** [comment]
closing as the parent PR was closed too as abandoned
